src/components/report-technical-issue/template.njk

```diff
@@ -0,0 +1,2 @@
+<a class="govuk-link {{ params.classList }}" target="_blank" href="/contact/problem_reports_nonjs?service={{ params.serviceCode | default("unknown") }}"
```

Comment:
- Have you thought about the change in behaviour with the form opening in a new tab?
- Generally we add some text in parenthesis to help users know that this is going to open in a new window or tab

Reply:
That's fair.  I'll update that.